sbpl_lattice_planner/src/sbpl_lattice_planner.cpp

```diff
@@ -42,6 +42,7 @@
 
 #include <costmap_2d/inflation_layer.h>
 #include <tf2/LinearMath/Quaternion.h>
+#include <tf/transform_broadcaster.h>
```

Comment:
I don't think we need to include the `transform_broadcaster` here. Maybe some more specific includes though? I haven't checked.